src/lib/market-service/coingecko/coingecko.ts

```diff
@@ -40,7 +48,7 @@ export class CoinGeckoMarketService implements MarketService {
     this.baseUrl = adapters.coingeckoProBaseUrl
   }
 
-  async findAll(args?: FindAllMarketArgs) {
+  async findAll(args?: FindAllMarketArgs, orderBy: CoinGeckoSortKey = 'market_cap_desc') {
```

Comment:
👀

Reply:
separated the orderBy since coingecko is the only provider supporting it